assembl/static2/js/app/reducers/adminReducer/index.js

```diff
@@ -198,6 +198,17 @@ export const discussionLanguagePreferencesHasChanged: DiscussionLanguagePreferen
   }
 };
 
+type ToggleLanguageMenu = boolean;
+type ToggleLanguageMenuReducer = (ToggleLanguageMenu, ReduxAction<Action>) => ToggleLanguageMenu;
+export const toggleLanguageMenu: ToggleLanguageMenuReducer = (state = false, action) => {
+  switch (action.type) {
+  case 'TOGGLE_LANGUAGE_MENU':
+    return action.state;
```

Comment:
if it is a toggle action, I think you should handle the fact that it toggles the value here (so no value in the action and `return !state`). If you need to be able to choose the value, I would rename the action `UPDATE_LANGUAGE_MENU_VISIBILITY` or something)

Reply:
ok